core/src/test/scala/org/broadinstitute/dsde/rawls/entities/local/EntityStatisticsCacheSupportSpec.scala

```diff
@@ -9,6 +9,7 @@ import scala.concurrent.ExecutionContext
 
 class CacheSupport(val dataSource: SlickDataSource, val workspaceContext: Workspace) extends EntityStatisticsCacheSupport {
   override implicit protected val executionContext: ExecutionContext = ExecutionContext.global
+  val workbenchMetricBaseName: String = "" // what to do?
```

Comment:
to do: should i set the base name via a parameter in the constructor? or is it constant